docs/proposed/psa-driver-interface.md

```diff
@@ -586,6 +632,14 @@ How does a driver author decide which location values to use? It should be possi
 
 Can the driver assembly process generate distinct location values as needed? This can be convenient, but it's also risky: if you upgrade a device, you need the location values to be the same between builds.
 
+The current plan is for Arm to maintain a registry of vendors and assign a location namespace to each vendor. Parts of the namespace would be reserved for implementations and integrators.
```

Comment:
minor: Given this seems to be a tentative plan, it might be worth keeping it in the "Open Questions" section. The main part of the spec should use strong language.

Reply:
This is in the open questions section.

Reply:
Sorry, I got a bit lost where I was looking